ipywidgets/interact.py

```diff
@@ -13,8 +19,11 @@ def _get_html(obj):
     if png_rep is not None:
         if isinstance(obj, plt.Figure):
             plt.close(obj)  # keep from displaying twice
-        return ('<img src="data:image/png;'
-                'base64,{0}">'.format(png_rep.encode('base64')))
+        # return ('<img src="data:image/png;'
+                # 'base64,{0}">'.format(png_rep.encode('base64'))) # fails in Python3
```

Comment:
This fails in Python 3. I used the base64 library instead. I believe that it is also compatible with Python 2,  according to the documentation at least.